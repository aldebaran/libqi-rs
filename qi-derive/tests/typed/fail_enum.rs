#[derive(qi_derive::Typed)]
enum E {
    A,
    B(i32),
    C { c: String },
}

fn main() {}
