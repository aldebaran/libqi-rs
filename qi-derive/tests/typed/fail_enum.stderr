error: `Typed` cannot be derived on enums and unions
 --> tests/typed/fail_enum.rs:2:6
  |
2 | enum E {
  |      ^
